parlai/agents/drqa/config.py

```diff
@@ -16,7 +16,7 @@ def add_cmdline_args(parser):
     # Basics
     agent.add_argument('--embedding_file', type=str, default=None,
                         help='File of space separated embeddings: w e1 ... ed')
-    agent.add_argument('--pretrained_model', type=str, default=None,
```

Comment:
why change this?

Reply:
the new train loop sets opt['init_model'] so that we can load from checkpoint, and @jaseweston suggested to name the parameter 'init_model' instead of 'pretrained_model'